### PR TITLE
Library upgrades to swagger, jackson and guava

### PR DESCRIPTION
**Pull request by Camelia-Orcid**

### CODE REVIEW

**Comment by Camelia-Orcid** [comment]
[Update jersey server to 3.0.5](https://trello.com/c/qNy3Usbd/8362-update-jersey-server-to-305)

File: src/main/java/org/orcid/jaxb/model/v3/release/record/CompletionDate.java [hunk 1]
```diff
@@ -15,8 +15,8 @@
 import javax.xml.bind.annotation.XmlValue;
 import javax.xml.datatype.XMLGregorianCalendar;
 
-import io.swagger.annotations.ApiModel;
-import io.swagger.annotations.ApiModelProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.media.Schema;
```
Comment (amontenegro) [inline src/main/java/org/orcid/jaxb/model/v3/release/record/CompletionDate.java:19]: Imported twice

File: src/main/java/org/orcid/jaxb/model/v3/release/record/Citation.java [hunk 2]
```diff
@@ -12,11 +12,11 @@
 import javax.xml.bind.annotation.*;
 import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
 
-import io.swagger.annotations.ApiModelProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
 import org.orcid.jaxb.model.common.CitationType;
 import org.orcid.jaxb.model.common.adapters.CitationTypeAdapter;
 
-import io.swagger.annotations.ApiModel;
+import io.swagger.v3.oas.annotations.media.Schema;
```
Comment (amontenegro) [inline src/main/java/org/orcid/jaxb/model/v3/release/record/Citation.java:19]: Imported twice, line 15

File: src/main/java/org/orcid/jaxb/model/v3/release/record/Affiliation.java [hunk 3]
```diff
@@ -17,12 +17,12 @@
 import org.orcid.jaxb.model.v3.release.common.Url;
 import org.orcid.jaxb.model.v3.release.common.Visibility;
 
-import io.swagger.annotations.ApiModel;
+import io.swagger.v3.oas.annotations.media.Schema;
```
Comment (amontenegro) [inline src/main/java/org/orcid/jaxb/model/v3/release/record/Affiliation.java:20]: Imported twice, line 9

File: src/main/java/org/orcid/jaxb/model/v3/release/record/Address.java [hunk 4]
```diff
@@ -16,15 +16,15 @@
...
+import io.swagger.v3.oas.annotations.media.Schema;
 import org.orcid.jaxb.model.v3.release.common.Country;
 import org.orcid.jaxb.model.v3.release.common.CreatedDate;
 import org.orcid.jaxb.model.v3.release.common.Filterable;
 import org.orcid.jaxb.model.v3.release.common.LastModifiedDate;
 import org.orcid.jaxb.model.v3.release.common.Source;
 import org.orcid.jaxb.model.v3.release.common.Visibility;
 
-import io.swagger.annotations.ApiModel;
+import io.swagger.v3.oas.annotations.media.Schema;
```
Comment (amontenegro) [inline src/main/java/org/orcid/jaxb/model/v3/release/record/Address.java:27]: Imported twice, line 19

File: src/main/java/org/orcid/jaxb/model/v3/release/notification/permission/Item.java [hunk 5]
```diff
@@ -19,12 +19,12 @@
...
 import javax.xml.bind.annotation.XmlType;
 
-import io.swagger.annotations.ApiModelProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
 import org.orcid.jaxb.model.common.ActionType;
 import org.orcid.jaxb.model.v3.release.record.ExternalID;
 import org.orcid.jaxb.model.v3.release.record.PersonExternalIdentifier;
 
-import io.swagger.annotations.ApiModel;
+import io.swagger.v3.oas.annotations.media.Schema;
```
Comment (amontenegro) [inline src/main/java/org/orcid/jaxb/model/v3/release/notification/permission/Item.java:27]: Imported twice, line 22

File: src/main/java/org/orcid/jaxb/model/v3/release/common/Year.java [hunk 6]
```diff
@@ -13,8 +13,8 @@
 import javax.xml.bind.annotation.XmlType;
 import javax.xml.bind.annotation.XmlValue;
 
-import io.swagger.annotations.ApiModel;
-import io.swagger.annotations.ApiModelProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.media.Schema;
```
Comment (amontenegro) [inline src/main/java/org/orcid/jaxb/model/v3/release/common/Year.java:17]: Imported twice

File: src/main/java/org/orcid/jaxb/model/v3/release/common/Month.java [hunk 7]
```diff
@@ -15,8 +15,8 @@
 
 import org.apache.commons.lang.StringUtils;
 
-import io.swagger.annotations.ApiModel;
-import io.swagger.annotations.ApiModelProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.media.Schema;
```
Comment (amontenegro) [inline src/main/java/org/orcid/jaxb/model/v3/release/common/Month.java:19]: Imported twice

File: src/main/java/org/orcid/jaxb/model/v3/release/common/LastModifiedDate.java [hunk 8]
```diff
@@ -15,8 +15,8 @@
 import javax.xml.bind.annotation.XmlValue;
 import javax.xml.datatype.XMLGregorianCalendar;
 
-import io.swagger.annotations.ApiModel;
-import io.swagger.annotations.ApiModelProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.media.Schema;
```
Comment (amontenegro) [inline src/main/java/org/orcid/jaxb/model/v3/release/common/LastModifiedDate.java:19]: Imported twice

**Review by amontenegro — CHANGES_REQUESTED** [review]

**Comment by Camelia-Orcid** [comment]
[Bring jax-b version in orcid-model up to date](https://trello.com/c/8WGd6hWz/8459-bring-jax-b-version-in-orcid-model-up-to-date)

**Comment by Camelia-Orcid** [comment]
[[dependabot] Jackson databind](https://trello.com/c/APbSUFst/8458-dependabot-jackson-databind)

**Comment by Camelia-Orcid** [comment]
[[QA] Remove release candidates from source code and Swagger](https://trello.com/c/zgjqmxCf/8124-qa-remove-release-candidates-from-source-code-and-swagger)

**Review by amontenegro — APPROVED** [review]

**Pull request merged**